tests/rules/no-missing-link-fragments.test.js

```diff
@@ -313,12 +310,13 @@ ruleTester.run("no-missing-link-fragments", rule, {
 			],
 		},
 
-		// Case-sensitive mismatch (without ignoreCase option)
+		// Case-sensitive mismatch (with ignoreCase option)
```

Comment:
```suggestion
		// Case-sensitive mismatch (with ignoreCase false option)
```

Could you add `false` to the comment for clearer description?